sympy/integrals/prde.py

```diff
@@ -986,18 +986,20 @@ def is_deriv_k(fa, fd, DE):
     dfa, dfd = dfa.cancel(dfd, include=True)
 
     # Our assumption here is that each monomial is recursively transcendental
-    if len(DE.L_K) + len(DE.E_K) != len(DE.D) - 1:
+    # TODO: probably need to change this later, since 'alg' is also contained
+    # in this now
+    if len(DE.exts) != len(DE.D):
```

Comment:
It is not necessary to test this at this point. Later, when the index lists for 'exp', etc. have been constructed we can raise NotImplementedError if the 'tan', 'atan' or 'alg' lists are nonempty. The test can then be edited after these parts are implemented.